.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - main
-    tags:
+    tags_ignore:
```

Comment:
perhaps this part should be completely removed, not sure

Reply:
Alternatively, we could remove the `release` below, no? Since when creating a release a tag is pushed anyway

Reply:
i think release published is needed since that's where it puts the assets. scalameta triggers just on tagging but it doesn't have assets.